Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -268,6 +313,8 @@ def is_tim_content(pr_files: list[str]) -> bool:
         if 'CONTRIBUTORS.json' in file:
             continue
         integration = BaseContent.from_path(CONTENT_PATH / file)
```

Comment:
let's change it to content_object since it is not an integration for sure
```suggestion
        content_object = BaseContent.from_path(CONTENT_PATH / file)
```